src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseViewInfo.cs

```diff
@@ -0,0 +1,21 @@
+
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+#nullable disable
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement
+{
+    /// <summary>
+    /// A class for storing various properties needed for initializing New Database dialog
```

Comment:
Why is this only for a new database? What about viewing properties? Or is new objects the only thing that these "view info" classes are for?

@alanrenmsft This is what I was mentioning to you, having clear documentation around what the base classes for these things are supposed to be used for would be great.

Reply:
Removed for now